yt/visualization/color_maps.py

```diff
@@ -508,7 +508,7 @@ def add_colormap(name, cdict):
 # Add colormaps in _colormap_data.py that weren't defined here
 _vs = np.linspace(0, 1, 256)
 for k, v in list(_cm.color_map_luts.items()):
-    if k not in yt_colormaps and k not in mcm.cmap_d:
+    if k not in yt_colormaps and k not in mcm._cmap_registry:
```

Comment:
From what I've seen this is indeed the new place where this dict lives in mpl as of 3.3, but do you know if it's backward compatible?

Reply:
fixed

Reply:
Well done ! I wonder if we should refactor to just NOT use this dict since they are not keeping it exposed... but this is out of scope here.